modules/tinymce/src/core/main/ts/paste/Paste.ts

```diff
@@ -19,6 +19,8 @@ const setup = (editor: Editor): void => {
   Commands.register(editor, pasteFormat);
   PrePostProcess.setup(editor);
 
+  editor.addQueryStateHandler('pasteasplaintext', () => pasteFormat.get() === 'text');
```

Comment:
I think the command state name should be the same as the command that alter it. But toggle makes it pretty odd though.

```suggestion
  editor.addQueryStateHandler('mceTogglePlainTextPaste', () => pasteFormat.get() === 'text');
```

Reply:
Yeah, I agree, putting it like that looks really weird. But it's mostly for internal use so we'll survive. I'll change it.